scripts/perftune.py

```diff
@@ -1305,6 +1305,13 @@ def parse_cpu_mask_from_yaml(y, field_name, fname):
     else:
         raise Exception("Bad '{}' value in {}: {}".format(field_name, fname, str(y[field_name])))
 
+def extend_and_unique(orig_list, iterable):
+    """
+    Extend items to a list, and make the list items unique
+    """
+    orig_list = orig_list.extend(iterable)
+    return list(dict.fromkeys(orig_list))
```

Comment:
list(set(orig_list))

Reply:
> list(set(orig_list))

Both of them work, what's the advantage of using set?